### DISCUSSION TITLE
Highlighting driver in all open windows when clicked

### DISCUSSION THREAD

**Discussion by rriegner** [f1multiviewer/issue-tracker#257] [Feature requests]
With so much data available, it would be nice to be able to highlight a driver to focus on them in all windows. For example, clicking on a driver in live timing highlights their background in the list, highlights or enlarges their marker on the map, highlights their telemetry bar on the OBC (or maybe a highlighted border if no telemetry). Maybe even highlighting race control messages pertaining to them.

Idea stolen from F1 Manager

**Comment by Jurtaani** [comment]
i would add focusing that highlighted driver would turn all other drivers in the live timing into +1.0 or -1.0 seconds ahead/behind that driver.

yes there already is info between two drivers and to leader, but i would like to for example know how far driver at 10th place is from driver in 6th place. also how their sector times compare.

**Comment by mmaarrtt** [comment]
I miss that opportunity too
